packages/lms/src/routes/search/+page.svelte

```diff
@@ -129,243 +125,181 @@
+									/>
+									<p>Lessons</p>
+								</label>
+							</div>
+						</div>
 					</div>
 				</div>
 			{/if}
-			<div>
-				<button class="show-option-btn" type="button" on:click={toggleOpen}
-					>Advanced Search Options</button
-				>
+			<div class="flex flex-wrap my-8 gap-4">
+				<button type="submit" class="btn variant-filled-primary" on:submit={handleSubmit}>Search</button>
+				<div>
+					<button class="btn variant-outline-primary variant-filled-secondary" type="button" on:click={toggleOpen}
+						>Advanced Search Options</button
+					>
+				</div>
 			</div>
-			<button type="submit" class="search-btn">Search</button>
 		</form>
 	</div>
 	<div>
 		{#if searchQuery.trim() !== ''}
-			<div class="bubble term">
-				<div class="key">{searchQueryExact ? 'Exact' : 'Fuzzy'}</div>
+			<div class="chip variant-filled">
+				<div class="key">{searchQueryExact ? 'Exact:' : 'Fuzzy:'}</div>
 				<div>{searchQuery}</div>
 			</div>
 		{/if}
 		{#if searchExclusions.length !== 0}
-			<div class="bubble exclusion">
+			<div class="chip variant-filled">
 				<div class="key">Excludes:</div>
 				<div>{searchExclusions}</div>
 			</div>
 		{/if}
 		{#if searchTags.length !== 0}
-			<div class="bubble tags">
+			<div class="chip variant-filled">
 				<div class="key">Tags:</div>
 				<div>{searchTags.join(', ')}</div>
 			</div>
 		{/if}
 		{#if searchTypes.length > 0}
-			<div class="bubble type">
+			<div class="chip variant-filled">
 				<div class="key">Types:</div>
 				<div>{searchTypes.join(', ')}</div>
 			</div>
 		{/if}
 		{#if selectedInstitution !== 'all'}
-			<div class="bubble institution">
+			<div class="chip variant-filled">
 				<div class="key">Institution:</div>
 				<div>{selectedInstitution}</div>
 			</div>
 		{/if}
 	</div>
 	{#if searchResults.length > 0}
-		<h2>Search Results</h2>
+		<h2 class="h2 mt-10 mb-8">Search Results</h2>
 	{:else}
-		<h2>No Results</h2>
+		<h2 class="h2 mt-10 mb-8">No Results</h2>
 	{/if}
 	<div class="results-container">
-		{#if searchResults}
-			{#each searchResults as result}
-				<div class="result">
-					<h3>{result.title}</h3>
-					<p>Search Score:{Math.round(result.score * 10000) / 10000}</p>
-					<p>Type: {result.type}</p>
-					<p>Tags: {result.tags}</p>
-					<a class="result-btn" href={result.browserPath}>View Details</a>
-				</div>
-			{/each}
+		{#if searchResults.length > 0}
+			<div class="grid sm:grid-cols-2 gap-4">
+				{#each searchResults as result}
+					<a class="card p-4 variant-ghost-surface no-underline dark:text-slate-100" href={result.browserPath}>
+						<header class="card-header border-b flex justify-between items-baseline gap-3 p-0">
+							<h3 class="h3">{result.title}</h3>
+							<span class="chip variant-filled">{result.type}</span>
+						</header>
+						<footer class="card-footer p-0 pt-2">
+							{#each result.tags as tag}
+								<p class="chip variant-ringed m-1 ms-0 pt-2">{tag}</p>
+							{/each}
+						</footer>
+					</a>
```

Comment:
Perhaps we should make a component for this as it needs to be reused on the tags page. Which currently is lacking styles on its cards now.

Reply:
Added